build/Repositories.props

```diff
@@ -6,49 +6,49 @@
-    <Repository Include="Hosting" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="HtmlAbstractions" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="HttpAbstractions" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="HttpSysServer" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Identity" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="IISIntegration" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="JavaScriptServices" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="JsonPatch" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="KestrelHttpServer" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Localization" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Logging" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MetaPackages" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Microsoft.Data.Sqlite" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MusicStore" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Mvc" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MvcPrecompilation" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Options" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Proxy" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Razor" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="ResponseCaching" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Routing" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Scaffolding" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Security" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="ServerTests" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Session" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="SignalR" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="StaticFiles" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Testing" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="WebSockets" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="Antiforgery" Commit="" CloneUrl="https://github.com/aspnet/%(Identity)" />
```

Comment:
We'll require fewer setup pieces on dev boxes to clone repositories (you cannot clone over ssh unless you have a key that's registered with github), but pushing would require having git credentials setup (which I haven't as yet because ssh). We used to be clever and try to clone over the protocol Universe was cloned in. Is that preferable?